src/js/utils/SystemLogUtil.js

```diff
@@ -0,0 +1,44 @@
+const SystemLogUtil = {
+  getUrl(nodeID, options, isStream = true) {
+    let {params = {}} = options;
+
+    let range = ['cursor', 'limit', 'skip_prev', 'skip_next'].reduce(function (memo, key) {
+      if (options[key] === '' || options[key] == null) {
+        return memo;
+      }
+      let rangeItem = `${key}=${encodeURIComponent(options[key])}`;
+      if (memo !== '') {
```

Comment:
Instead of reducing on a string, we could reduce onto an Array - `memo.push(rangeItem)` and then after the reduce function has completed we can do `range.join('&')`. This circumvent this `if (memo !== '') {}` check. It's also much more efficient (although for a small number of iterations it doesn't really matter).